mite_browser/__init__.py

```diff
@@ -409,8 +409,11 @@ def __repr__(self):
 
 
 def _field_is_disabled(element):
-    status = element.get('disabled')
-    return status is not None and status.lower() in ['disabled', 'true']
+    status = element.attrs.get('disabled')
+    if status and status.lower() in ['disabled', 'true']:
```

Comment:
Bit of a nit. Not sure if you have a preference but I find this is a trap people fall in to without even noticing. Maybe just 

```
`return status and status.lower() in ['disabled', 'true']
```